docs/service/artifact-registry.md

```diff
@@ -1,9 +1,9 @@
-# The Artifact Registry
+# The Artifact registry
 
-The Artifact Registry serves up Artifact files. You can list the current set of Artifacts and their versions
+The Artifact registry serves up Artifact files. You can list the current set of Artifacts and their versions
```

Comment:
The Artifact Registry is a discrete thing. 'Registry' should have the same capitalization as 'Artifact', whatever that is.